### ISSUE TITLE
Cross switch

### ISSUE THREAD

**Issue by m42e** [boaschti/MQTT_Mini_WLan_Relais#1]
Ever thought of a cross switch Relais with such a board? Thanks.

**Comment by boaschti** [comment]
Of course. Just connect NO NC COM of the relay and 230Vin to readback the lamp state. Use this board: https://github.com/boaschti/MQTT_Mini_WLan_Relais/tree/master/Board/Wlan%20Relais
I use Tasmota Firmware with a modification. See: https://github.com/arendst/Sonoff-Tasmota/issues/1253 and to configure see: https://github.com/arendst/Sonoff-Tasmota/issues/1216
